lib/ansible/modules/user.py

```diff
@@ -449,6 +460,11 @@
   returned: When user exists
   type: int
   sample: 20
+password_expire_warn:
```

Comment:
Unless we're missing something, it looks like this return value is never set- since this isn't supported everywhere, maybe better to just remove it...

Reply:
Thanks for your feedback @nitzmahone! I removed it as suggested.

What do you think about how to proceed with the `password_expire_max` and `password_expire_min` return values since they are also never set? Remove them in a separate PR?

Reply:
Yeah, we noticed that as well while looking this over- as far as I'm concerned just removing them in this PR is fine, since you're addressing things very close to that neighborhood anyway...

Reply:
All right, cleaned it up.

Reply:
Ok, meanwhile the wrong return docs were removed in a separate PR #79891.